script/component-migration/transformers/additionalinfo.js

```diff
@@ -0,0 +1,27 @@
+function additionalInfoReplacement() {
+  /**
+   * Remove prop from component if it exists.
+   *
+   * @param {string} componentString A string containing an occurrence of a component
+   * @param {string} propName The name of the prop to be removed
+   */
+  const removeProp = (componentString, propName) => {
+    const regex = new RegExp(`(${propName}=["{](.)+?["}])`);
+    const propExists = componentString
+      .substring(0, componentString.indexOf('>'))
+      .includes(propName);
+    return propExists ? componentString.replace(regex, '') : componentString;
+  };
+
+  return [
+    'va-additional-info',
+    {
+      className: removeProp,
```

Comment:
Let's leave `className` alone for now. We don't want to remove utility classes that people are intentionally adding, and the work to conditionally remove `className` based on which classes are inside it are out of scope for this PR. We should probably have a function that we can import like `removeCustomClasses` eventually.

![image](https://user-images.githubusercontent.com/2008881/146077720-38e3ab9e-ee7c-4f0c-b1fd-39899a170f05.png)